bin/setup

```diff
@@ -4,8 +4,13 @@ require 'pathname'
 gem_root = Pathname.new(__dir__).join("..")
 
 unless gem_root.join("spec/manageiq").exist?
-  puts "== Cloning manageiq sample app =="
-  system "git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 spec/manageiq"
+  if ENV.key?("MANAGEIQ_REPO")
+    puts "== Symlinking #{ENV["MANAGEIQ_REPO"]} to spec/manageiq"
```

Comment:
I think the wording here is backwards... Should it be 

puts "== Symlinking spec/manageiq to #{ENV["MANAGEIQ_REPO"]}"

Reply:
You're right, fixed